app/controllers/workspaces_controller.rb

```diff
@@ -76,8 +76,36 @@ def destroy
     end
   end
 
+  # POST /workspaces/1/favorite
+  def favorite
+    params.permit(:favorite, :id)
```

Comment:
This is kinda ugly and for some reason (haven't looked into it yet), an unused and unpermitted `:workspace` param is coming through. not sure why, but we don't use it. rails machinery?